### PR TITLE
Update the propTypes to reuse the Minicart's one

### PR DESCRIPTION
**Pull request by vitoria**
#### What is the purpose of this pull request?
Update the propTypes to reuse the mini cart's one.

#### What problem is this solving?
This component was showing lots of warnings in the console because there were some incompatibilities with the propTypes.

#### How should this be manually tested?
(Access the workspace)[https://waza--storecomponents.myvtex.com/]

#### Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/MiniCartContent.js [hunk 1]
```diff
@@ -19,15 +19,16 @@ class MiniCartContent extends Component {
   static propTypes = {
     /* Products in the cart */
     data: MiniCartPropTypes.data,
-    /* Mutate function */
-    mutate: PropTypes.func.isRequired,
     /* Label to appear when the minicart is empty */
```
Comment (salesfelipe) [inline react/MiniCartContent.js]: Label that will appear...

File: react/MiniCartContent.js [hunk 2]
```diff
@@ -19,15 +19,16 @@ class MiniCartContent extends Component {
...
-    labelMiniCartEmpty: PropTypes.string,
+    labelMiniCartEmpty: MiniCartPropTypes.labelMiniCartEmpty,
     /* Label to appear in the finish shopping button */
```
Comment (salesfelipe) [inline react/MiniCartContent.js]: idem

File: react/MiniCartContent.js [hunk 3]
```diff
@@ -19,15 +19,16 @@ class MiniCartContent extends Component {
...
-    labelButton: PropTypes.string,
+    labelButton: MiniCartPropTypes.labelButtonFinishShopping,
     /* Show remove item button or not */
-    showRemoveButton: PropTypes.bool,
+    showRemoveButton: MiniCartPropTypes.showRemoveButton,
+    /* Function to be called when a item is removed */
```
Comment (salesfelipe) [inline react/MiniCartContent.js]: when an item

File: react/MiniCartPropTypes.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ export const MiniCartPropTypes = {
   /* Mini cart icon color */
   miniCartIconColor: PropTypes.string,
   /* Show the remove item button or not */
```
Comment (brunojdo) [inline react/MiniCartPropTypes.js]: Set the visibility of remove button

**Review by salesfelipe — CHANGES_REQUESTED** [review]

**Review by claudivanfilho — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Changed
 - Change the mini cart components to use the store components.
+- Change PropTypes to reuse the mini cart's one.
```
Comment (lucasecdb) [inline CHANGELOG.md]: No need to place `Change...` in the beginning, just describe what was changed.

**Review by lucasecdb — CHANGES_REQUESTED** [review]

File: react/MiniCartContent.js (hunk 1 above)
Comment (vitoria) [reply react/MiniCartContent.js]: DOne!

File: react/MiniCartContent.js (hunk 2 above)
Comment (vitoria) [reply react/MiniCartContent.js]: Done!

File: react/MiniCartContent.js (hunk 3 above)
Comment (vitoria) [reply react/MiniCartContent.js]: Done!

**Review by lucasecdb — APPROVED** [review]
I think the alias prop types you define (such as `MiniCartPropTypes.labelMiniCartEmpty`) are unnecessary, and you should only define ones that are a complex type (e.g. `MiniCartPropTypes.data`), but it's something to discuss later.

**Review by salesfelipe — APPROVED** [review]

File: react/MiniCartContent.js [hunk 6]
```diff
@@ -17,17 +17,14 @@ import './global.css'
  */
 class MiniCartContent extends Component {
   static propTypes = {
-    /* Products in the cart */
     data: MiniCartPropTypes.data,
+    labelMiniCartEmpty: MiniCartPropTypes.labelMiniCartEmpty,
```
Comment (brunojdo) [inline react/MiniCartContent.js]: What's new put at the beginning of propTypes

Comment (brunojdo) [reply react/MiniCartContent.js]: Add a comment /* Reused props */

**Review by brunojdo — APPROVED** [review]

File: react/MiniCartPropTypes.js (hunk 4 above)
Comment (vitoria) [reply react/MiniCartPropTypes.js]: Done!

**Pull request merged**